includes/engelsystem_provider.php

```diff
@@ -77,7 +78,7 @@
 require_once realpath(__DIR__ . '/../vendor/parsedown/Parsedown.php');
 
 if (! defined('PHPUNIT_TESTSUITE')) {
-  session_lifetime(24 * 60, preg_replace("/[^a-z0-9-]/", '', md5(__DIR__)));
+  //session_lifetime(24 * 60, preg_replace("/[^a-z0-9-]/", '', md5(__DIR__)));
```

Comment:
Please pull or fetch the current state of the engel_muenchen branch and do a rebase on it.